akka-docs/src/test/java/jdocs/actor/ActorDocTest.java

```diff
@@ -581,6 +581,7 @@ public void using_receiveTimeout() {
         myActor.tell("Hello", getRef());
         expectMsgEquals("Hello world");
         expectMsgEquals("timeout");
+        expectNoMessage(Duration.ofSeconds(2));
```

Comment:
add this method to test will not receive Timeout after cancel the receive timeout.

Reply:
`expectNoMessage` will always wait for the full duration, a permanent 2 seconds is a lot to add to a test, would be better create a separate test with a shorter timeout for this, but otoh it is perhaps too easy to cause a race condition in the test with a short timeout (cancelling only prevents future receive timeout, not one that already triggered but was not yet processed by the actor)